.github/workflows/deploy_to_aws.yml

```diff
@@ -261,6 +261,43 @@ jobs:
           string: ${{ vars.ENV_NAME }}
       - name: Checkout repo
         uses: actions/checkout@v4
+      - name: Stop Logstash for ${{ vars.ENV_NAME }}
+        if: ${{ (vars.ENV_NAME != 'Production' && vars.ENV_NAME != 'Staging' && vars.ENV_NAME) && inputs.restore-db == true}}
+        run: |
+          docker run --rm \
+              -e AWS_ACCESS_KEY_ID=${{ secrets.AWS_ACCESS_KEY_ID }} \
+              -e AWS_SECRET_ACCESS_KEY=${{ secrets.AWS_SECRET_ACCESS_KEY }} \
+              -e AWS_DEFAULT_REGION=eu-west-1 \
+              amazon/aws-cli \
+              ecs update-service --desired-count 0 --cluster=ecsOpenSupplyHub${{vars.ENV_NAME}}Cluster \
+              --service=OpenSupplyHub${{vars.ENV_NAME}}AppLogstash  --region=eu-west-1
+      - name: Clear OpenSearch index for ${{ vars.ENV_NAME }}
+        if: ${{ (vars.ENV_NAME != 'Production' && vars.ENV_NAME != 'Staging' && vars.ENV_NAME) && inputs.restore-db == true}}
+        run: |
+          mkdir -p script
+          mkdir -p ssh
+          echo -e "Host *\n\tStrictHostKeyChecking no\n\n" > ssh/config
+          printf "%s\n" "${{ secrets.SSH_PRIVATE_KEY }}" > ssh/id_rsa
+          echo "" >> ssh/id_rsa
+          echo -n ${{ vars.BASTION_IP }} > script/.env
+          cat <<EOF > script/clear_opensearch.sh
+          #!/bin/bash
+          curl -X DELETE https://${{ vars.OPENSEARCH_DOMAIN }}/production-locations --aws-sigv4 "aws:amz:eu-west-1:es" --user "${{ secrets.AWS_ACCESS_KEY_ID }}:${{ secrets.AWS_SECRET_ACCESS_KEY }}"
```

Comment:
Is OPENSEARCH_DOMAIN a static value even if the environment is recreated like pre-prod?

Reply:
OpenSearch domain won't be changed when redeployed.

Reply:
Even if the environment is recreated from scratch?

Reply:
I will add getting domain url by was cli